web/src/server/api/services/tableDefinitions.ts

```diff
@@ -114,6 +115,14 @@ export const calculatedTotalCost = {
   internal: 'o."calculated_total_cost"',
 } as const;
 
+export const traceTags = {
+  name: "Tags",
+  id: "tags",
+  type: "arrayOptions",
+  options: [] as OptionsDefinition[],
```

Comment:
what do we need these options for?

Reply:
traceTags needs to be of type ColumnDefinition. So far we declared tags columnDefinitions as type arrayOptions, therefore I thought we should continue doing this. But in this case it only adds overhead and does not change the functionality. Good point, I changed it to type string and this enables us to remove the options.